include/itkRunLengthTextureFeaturesImageFilter.h

```diff
@@ -191,6 +193,8 @@ class ITK_TEMPLATE_EXPORT RunLengthTextureFeaturesImageFilter:public ImageToImag
    * Set the min and max (inclusive) pixel value that will be used in
    * generating the histogram.
    */
+  itkSetMacro( MinDistance, RealType);
+  itkSetMacro( MaxDistance, RealType);
```

Comment:
Rename these to DistanceMinimum and DistanceMaximum too?

Reply:
Yes, `MinimumDistance`, `MaximumDistance`, please.